src/Form/LocalistEntityForm.php

```diff
@@ -240,13 +240,13 @@ public function form(array $form, FormStateInterface $form_state) {
       '#required' => FALSE,
     ];
 
-    $form['localist_image_field_name'] = [
+    $form['localist_media_field_name'] = [
       '#type' => 'textfield',
-      '#title' => $this->t('Localist event image field'),
-      '#default_value' => $localist_pull->localist_image_field_name,
+      '#title' => $this->t('Localist event media field'),
+      '#default_value' => $localist_pull->localist_media_field_name,
       '#size' => 20,
       '#maxlength' => 255,
-      '#description' => $this->t('Mapping: field machine name for the localist image'),
+      '#description' => $this->t('Mapping: field machine name for the localist media'),
```

Comment:
Woah, it took me WAY too long to figure out that we have this field in here twice haha, I was like, "he's replacing it, but also it's still there, what is going onnnnnnnnn" -- glad that this work will have the side benefit of removing the duplicated form field declaration! 🎉